### ISSUE TITLE
Inspector - json is not showing error messages correctly

### ISSUE THREAD

**Issue by EricDahlvang** [microsoft/BotFramework-Emulator#1486]
## Version
Nightly: 4.3.3-57327

## Describe the bug
Clicking a **500** in the LOG, and the inspector separates every letter by line:
```json
{
  "0": "<",
  "1": "!",
  "2": "D",
  "3": "O",
  "4": "C",
  "5": "T",
  "6": "Y",
  "7": "P",
  "8": "E",
  "9": " ",
  "10": "h",
```

## Screenshots
![image](https://user-images.githubusercontent.com/11055362/57047997-43e33980-6c27-11e9-92de-b031472548ef.png)

[bug]

**Comment by justinwilaby** [comment]
Looks like the 500 is returned by the server as an HTML document with a `content-type: application/json` header. The Emulator is very likely displaying this correctly. The server seems to be setting the wrong headers. I'll investigate...

**Comment by justinwilaby** [comment]
I'm not able to repro this and am nearly certain this is not a problem with the emulator. My first comment is likely the reason this is seen. Once the server sends the appropriate content and content-type header with the response, the error will likely be displayed correctly. On a side note, HTML responses are not supported within the emulator's json viewer and are not expected as a response from a 500 error.

<img width="1403" alt="image" src="https://user-images.githubusercontent.com/2652885/57539696-e1adc700-72ff-11e9-903b-d9b242f6a7cd.png">

**Comment by corinagum** [comment]
I'm reproing this:
![image](https://user-images.githubusercontent.com/14900841/57638415-91826f00-7562-11e9-967a-f6b9043b6a96.png)

**Comment by justinwilaby** [comment]
@corinagum - What you're seeing was introduced [here](https://github.com/microsoft/BotFramework-Emulator/pull/1522/files#diff-94c1049e84163f209cd15131302c7a8aR245)
@cwhitten - Is this intended behavior?

**Comment by cwhitten** [comment]
I've got this to clean this up but maintain what I wanted to provide in certain error conditions: https://github.com/microsoft/BotFramework-Emulator/pull/1570

**Comment by tonyanziano** [comment]
I believe this was closed by #1570
